api/apps/api/src/modules/access-control/projects-acl/project-acl.controller.ts

```diff
@@ -0,0 +1,56 @@
+import { apiGlobalPrefixes } from '@marxan-api/api.config';
+import {
+  Controller,
+  Get,
+  Req,
+  Param,
+  ParseUUIDPipe,
+  UseGuards,
+  Delete,
+  Patch,
+  Body,
+} from '@nestjs/common';
+import { JwtAuthGuard } from '@marxan-api/guards/jwt-auth.guard';
+import { ProjectAclService } from './project-acl.service';
+import { RequestWithAuthenticatedUser } from '@marxan-api/app.controller';
+import { ApiBearerAuth, ApiTags } from '@nestjs/swagger';
+import { UserRoleInProjectDto } from './dto/user-role-project.dto';
+
+@UseGuards(JwtAuthGuard)
+@ApiBearerAuth()
+@ApiTags('Projects-Users Roles')
+@Controller(`${apiGlobalPrefixes.v1}/roles/projects`)
+export class ProjectAclController {
+  constructor(private readonly projectAclService: ProjectAclService) {}
+
+  @Get(':projectId/users')
+  async findUsersInProject(
+    @Param('projectId', ParseUUIDPipe) projectId: string,
+    @Req() req: RequestWithAuthenticatedUser,
+  ): Promise<UserRoleInProjectDto[]> {
+    await this.projectAclService.checkUserIsOwner(req.user.id, projectId);
```

Comment:
I think what Kamil meant in [this comment](https://github.com/Vizzuality/marxan-cloud/pull/640#discussion_r752952683) - and I agree - is that we should move the `checkUserIsOwner` call to **inside** the service methods themsleves (like, e.g. inside `findUsersInProject`).

With this, we are making sure that every time `findUsersInProject` is called, the `checkUserIsOwner` validation is performed without the caller having to worry about this :) plus, the controller becomes even cleaner :ok_hand:

Reply:
Oh damn, good idea, but I didn't understood it well. Thanks for pointing it out!